server/scripts/seed.ts

```diff
@@ -0,0 +1,36 @@
+import Db from 'server/lib/db';
+import log from 'server/lib/log';
+import User from 'server/models/User.model';
+import { makeUser } from 'server/test/utils';
+import { UserObject } from 'server/types/User';
+
+async function create100Users(): Promise<UserObject[]> {
+  const promises = Array.from({ length: 100 }, makeUser);
+  const userArray = await Promise.all(promises);
+  return userArray;
+}
```

Comment:
1. To keep consistent verb usage, I'd name this `make100Users()` since it only builds the objects, whereas `create` would be inserting them.

2. `makeUser` is not an `async` function and therefore shouldn't require being wrapped in `Promise.all()`.

3. I agree, this would be useful in testing if it were more dynamic, perhaps naming it just `makeUsers`. In which case, you could probably move it into the `Utils` file next to `makeUser()` and just import it.

```suggestion
function makeUsers(count = 100): UserObject[] {
  return Array.from({ length: count }, makeUser);
}
```